controllers/nexus/resource/validation/validation.go

```diff
@@ -84,37 +82,37 @@ func (v *Validator) validate(nexus *v1alpha1.Nexus) error {
 
 func (v *Validator) validateNetworking(nexus *v1alpha1.Nexus) error {
 	if !nexus.Spec.Networking.Expose {
-		v.log.Debugf("'spec.networking.expose' set to 'false', ignoring networking configuration")
+		v.log.Debug("'spec.networking.expose' set to 'false', ignoring networking configuration")
 		return nil
 	}
 
 	if !v.ingressAvailable && nexus.Spec.Networking.ExposeAs == v1alpha1.IngressExposeType {
-		v.log.Errorf("Ingresses are not available on your cluster. Make sure to be running Kubernetes > 1.14 or if you're running Openshift set 'spec.networking.exposeAs' to '%s'. Alternatively you may also try '%s'", v1alpha1.IngressExposeType, v1alpha1.NodePortExposeType)
+		v.log.Info("Ingresses are not available on your cluster. Make sure to be running Kubernetes > 1.14 or if you're running Openshift set ", "spec.networking.exposeAs", v1alpha1.IngressExposeType, "Also try", v1alpha1.NodePortExposeType)
```

Comment:
Those are not errors since we don't have an error instance. We can consider review the validation approach (#174), or just raise a `errors.New()` here.

Reply:
Ah, I see. #174 looks great, but in the meantime I think we should go for `errors.New()` here with this message.

Reply:
I believe here we can go either with `errors.New()` or `log.Warn()`, I'll take a closer look to it.

Reply:
I believe error would be better, this leads to a reconcile error. When we have validation webhooks this will even prevent the user from storing the resource altogether. A warning implies the standard flow will be followed (though possibly with odd behavior), which isn't the case.

Reply:
Changed o `Warn` since in the next line we do raise the actual error.